static/app/utils/metrics/useMetricsMeta.tsx

```diff
@@ -48,7 +48,12 @@ export function useMetricsMeta(
   });
 
   return {
-    data: [...sessionMeta, ...txnsMeta, ...customMeta, ...spansMeta],
+    data: [
+      ...(enabledUseCases.includes('sessions') ? sessionMeta : []),
+      ...(enabledUseCases.includes('transactions') ? txnsMeta : []),
+      ...(enabledUseCases.includes('custom') ? customMeta : []),
+      ...(enabledUseCases.includes('spans') ? spansMeta : []),
+    ],
```

Comment:
This used to be like this, i refactored it a couple of weeks ago since it should be enough to pass the enabled flag to react-query. What did i miss? 😅

Reply:
Sometimes, when I visited settings from the DDM page, I would also get served other use cases, not just the custom that I asked for. Maybe it's enough to just pass a different cache key or something - let's talk tomorrow.